r-pkg/R/elasticsearch_parsers.R

```diff
@@ -804,6 +804,11 @@ es_search <- function(es_host
     
     # If we got everything possible, just return here
     hits_pulled <- length(firstResult[["hits"]][["hits"]])
+    
+    if (hits_pulled == 0) {
+      return(NULL)
```

Comment:
I think we need to add a warning here. Something like "Your query was syntactically valid but matched 0 documents. Returning NULL."

Thoughts @austin3dickey ?